kernel-kit/configs_x86/DOTconfig-6.1.52-x86

```diff
@@ -0,0 +1,8012 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Linux/x86 6.1.x Kernel Configuration
+#
+CONFIG_CC_VERSION_TEXT="gcc (GCC) 11.2.0"
```

Comment:
I'd heavily recommend you to run `make menuconfig` to re-generate this configuration file from Debian 12. You're going to build this kernel under a Debian 12 container (that's what you put in kernel-kit.yml) but you configured the kernel in a different distro (I can tell by the older GCC version), so it's not guaranteed the build result matches the configuration file (because it will pick default values for configuration keys not available under your build environment).